### DISCUSSION TITLE
MICROSOFT STREAM

### DISCUSSION THREAD

**Discussion by Moratech** [Launch-X-Latam/MisionFrontEnd#113] [General]
QUISIERA SABER SI ALGUIEN MAS TIENE INCONVENIENTES EN INICIAR SESION EN MICROSOFT STREAM  ME APARECE COMO CARGANDO PERO NUNCA SE ACTUALIZA LA PESTAÑA

**Comment by GustavoLoreto** [comment]
Hola!
Tengo exactamente el mismo problema con la app de escritorio, solo se queda en el logo de Teams y no pasa de ahí.
En la versión web tampoco puedo ingresar, tengo el siguiente error en pantalla.
![image](https://user-images.githubusercontent.com/96457266/193425982-69d2877e-b369-4a12-83b2-e3e839f880f6.png)

**Reply by Moratech** [reply]
YO QUIERO VER LAS GRABACIONES DE LAS CLASES EN MICROSOFT STREAM Y NO ME DEJA INICIAR SESION SOLAMENTE SE QUEDA ACTUALIZANDO NO SE SI ES UN PROBLEMA DE LA PLATAFORMA O MI INTERNET

**Reply by CarloAyala** [reply]
@Moratech  
Aquí esta la ultima: [Ultima sesión](https://web.microsoftstream.com/video/41b9721d-cde1-4ffd-bdf0-7a877d57e5e1)
